src/client/java/net/neoforged/neoforge/client/event/EntityRenderersEvent.java

```diff
@@ -251,5 +254,11 @@ public void registerSkullModel(SkullBlock.Type type, Function<EntityModelSet, Sk
                 throw new IllegalArgumentException("Factory already registered for provided skull type: " + type.getSerializedName());
             }
         }
+
+        public void registerSkullBlockTexture(SkullBlock.Type type, ResourceLocation texture) {
+            if (skinByType.putIfAbsent(type, texture) != null) {
+                throw new IllegalArgumentException("Texture already registered for provided skull type: " + type.getSerializedName());
+            }
+        }
```

Comment:
Is there a reason this should be a separate method rather than being an overload of `registerSkullModel`? From what I can tell, there is a one-to-one mapping from `SkullBlock$Type` to model and texture. So, unless you are some mod, the vanilla instance needs both the model and the texture to render properly.